_projects/2018/plasmapy/plasma-object.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Plasma class
+desc: Devlop a new Plasma class for loading and storing multiple types of plasma physics data.
+requirements:
+ - Good knowledge of Python, including object-oriented programming
+ - Familiarity with basics of plasma physics
+ - Some experience with plasma physics data formats
+difficulty: High
+issues:
+ - 
+mentors:
+ - SolarDrew
+initiatives:
+ - GSOC
+tags:
+ - python
+ - 
+collaborating_projects:
+ - PlasmaPy
+---
+
+#### Description
+
```

Comment:
We should probably add a description of what PlasmaPy is and why it's awesome. :smile_cat:

Reply:
...and also say that plasma physics is important for fusion power, in addition to plasma astrophysics, solar physics, space physics, laboratory plasma physics, etc.

Reply:
True. I think I assumed that would be somewhere else, perhaps on the main OpenAstronomy GSOC page. Is that right @DavidPS, or have I made that up?

Reply:
Yes, plasmapy is in the members page.

Reply:
(though it doesn't say about fussion power, or any other application)